docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1096,3 +1098,56 @@ function su_humsci_profile_update_9718() {
   // Uninstalls modules after removing configs to prevent updb errors.
   \Drupal::service('module_installer')->uninstall(['hs_webform', 'webform_ui', 'webform']);
 }
+
+/**
+ * Add new logout block to existing sites.
+ */
+function su_humsci_profile_update_9719() {
+  $config_storage = new FileStorage(Settings::get('config_sync_directory'));
```

Comment:
For this one, I did try that initially, but it fails:
`>  [error]  The configuration <em class="placeholder">block.block.humsci_colorful_samlsunetidlogoutblock</em> was not found in the sync directory.`
I found what worked here: https://github.com/SU-HSDO/suhumsci/blob/416ecc8ccfedb3e759e7c91e129a92b8acd7e363/docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install#L19

My theory is that this is dealing with theme config? But not sure if that is correct. If we're set on using the service, I'll have to poke at that again next week.

Reply:
That's weird. We have both versions in the site, but in the past we've had kickbacks because SWS prefers to use the service. Let's keep the `FileStorage()` version for now and come back if needed after SWS review.

Reply:
`\Drupal::service('config.storage.sync')` works fine for me. Using the service is preferred.